### PR TITLE
iOS input updates

### LINKED ISSUE #5023: [iOS][ChoiceSet] [choice set not rendered properly for long text]
# Platform

What platform is your issue or question related to? (Delete other platforms).

- [ ]  iOS

# Author or host

Teams 

# Version of SDK

2.3.0

# Details
Choice set text is getting clipped and choices are getting animated on selection. Tried removing the custom choice set renderer and found out issue is there with SDK itself.
Issue is reproing in sample app with this json  [inp.txt](https://github.com/microsoft/AdaptiveCards/files/5484789/inp.txt)

Sample screenshots
| |

### LINKED ISSUE #5024: [iOS][Accessibility] [View is getting scrolled on submit validation for input.text and input.number when voice over is on]
# Platform

- [ ]  iOS

# Author or host
Teams

# Version of SDK
2.3.0

# Details
View is getting scrolled and hence, focus is not going to the accessibility element when we have input.text / input.number as the first failed input.

### PR DESCRIPTION
**Pull request by jwoo-msft**
## Related Issue
Fixed #5023, Fixed #5024 

## Description
Input.ChoiceSet had a regression in its layout; reverted ChoiceSet Xib related update
Setting focus on Input.Text and Input.Number caused erratic scrolling behavior. Updated to have the focus to be set on InputView of input, and allowed the scrolling to be handled by the host app, and explicitly set the accessibility focus on the input handler to announce the accessibility l

## Sample Card
https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.3/Elements/Input.ChoiceSet.ErrorMessage.json
https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.3/Elements/Input.Number.Label.json
https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.3/Elements/Input.Text.Label.json

## How Verified
How you verified the fix, including one or all of the following: 
1. Run most of the cards in 1.0/1.1/1.2 cards and run all of 1.3/elements

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5030)

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnSetRenderer.mm [hunk 1]
```diff
@@ -47,6 +47,7 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
                                                                   parentStyle:[viewGroup style]
                                                                    hostConfig:acoConfig
                                                                     superview:viewGroup];
```
Comment (shalinijoshi19) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnSetRenderer.mm:49]: Seems to be an unnecessary whitepace/newline change that can be reverted.

**Comment by shalinijoshi19** [comment]
What was the "xib related update for choiceset that we needed to revert?"

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm [hunk 2]
```diff
@@ -262,7 +291,7 @@ - (NSString *)getTitlesOfChoices
 
 - (float)getNonInputWidth:(UITableViewCell *)cell
 {
-    return cell.separatorInset.left + cell.indentationWidth + cell.accessoryView.frame.size.width + cell.imageView.image.size.width;
+    return padding * 3 + cell.imageView.image.size.width;
```
Comment (shalinijoshi19) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm:294]: > padding * 3 +  [](start = 10, length = 15)

What is the magic number here?

Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm:294]: it's the padding that's put before the cell's image, the front of the row, and the end of the row.

**Review by jwoo-msft — COMMENTED** [review]
> What was the "xib related update for choiceset that we needed to revert?"

I've added an additional xib for choice set for customization for v1.3.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnSetRenderer.mm (hunk 1 above)
Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnSetRenderer.mm:49]: the change is added by clang-format that format the source.

**Review by shalinijoshi19 — APPROVED** [review]

**Pull request merged**